packages/components/src/utils/array-utils.test.ts

```diff
@@ -13,4 +13,37 @@ describe('ArrayUtils', () => {
       expect(newArray).not.toEqual(inputArray);
     });
   });
+
+  describe('removeFromArray', () => {
+    it('should remove first item of array', () => {
+      const array = Object.freeze([0, 1, 2, 3]);
+      const { item, rest } = removeFromArray(array, 0);
+      expect(item).toBe(0);
+      expect(rest).toEqual([1, 2, 3]);
+    });
```

Comment:
No big deal but I think you should add an empty line between each test. It makes it more readable 😄